opencti-platform/opencti-graphql/src/utils/sorting.ts

```diff
@@ -0,0 +1,35 @@
+import { isDateNumericOrBooleanAttribute, schemaAttributesDefinition } from '../schema/schema-attributes';
+import { UnsupportedError } from '../config/errors';
+
+export const buildElasticSortingForAttributeCriteria = (orderCriteria: string, orderMode: 'asc' | 'desc') => {
```

Comment:
the algorithm here is a refactoring, it shall cover all previous cases properly and add the object case with sortBy.